getting-started/templates/systemlink-secrets.yaml

```
@@ -270,6 +270,9 @@ fileingestion:
     ## Cryptographic key used for AES-256 encryption of data at restI. Use a 32-byte cryptographically random value which is base64 encoded.
```

```suggestion
    ## Cryptographic key used for AES-256 encryption of data at rest. Use a 32-byte cryptographically random value which is base64 encoded.
```

Can we fix this typo while we're here?

---

Sure